minecode/miners/pypi.py

```diff
@@ -259,17 +259,28 @@ def build_packages(metadata, purl=None):
         if not url:
             continue
 
+        packagetype = None
```

Comment:
@chinyeungli The changes you made here are breaking the tests in for `minecode/tests/miners/test_pypi.py` https://dev.azure.com/nexB/purldb/_build/results?buildId=15751&view=logs&j=0cfca3bc-aa91-566e-292e-f125b0e3456a&t=0ab68744-bd25-5ccd-cfbc-bd1d5e7178c3&l=1033